### DISCUSSION TITLE
Testimonial section

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#144] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/162663376-d575d679-18fb-452c-b164-9fac944df95f.png)

The images on the bottom line are being cropped. 

"Anonymous" in the testimonial can be replaced by the related project

**Comment by an-nadaa** [comment]
That's the actual design. images are cropped by design. as for the anonymous. i prefer to keep it to show that we do care about preserving the dignity of our beneficiaries. and adding the project name will become very lengthy.
